src/i18n/en.json

```diff
@@ -1,188 +1,186 @@
-    "approverEmail": "Email",
-    "approverPhone": "Phone"
+  "LANGUAGE_OPTIONS": {
+    "ARABIC": "Arabic",
+    "ENGLISH": "English",
+    "ESTONIAN": "Estonian",
+    "FINNISH": "Finnish",
+    "RUSSIAN": "Russian",
+    "SOMALI": "Somali",
+    "SWEDISH": "Swedish"
   },
-  "nav": {
-    "signin": "Log in",
-    "signout": "Log out",
-    "menuButtonLabel": "Profile menu"
+  "loading": "Loading...",
+  "login": {
+    "buttonText": "Get membership",
+    "findNearestService": "Find the nearest youth centre on the service map",
+    "helpText": "Jässäri gives you access to the Youth Services’ activities. With Jässäri membership, you can participate in fun activities, hang out at Youth Centres, join hobby groups, courses and camps, apply for project subsidies or even produce your own event! The activities at Youth Centres are instructed by trained youth workers.",
+    "helpTextUnderAge": "Unfortunately, you need to be at least 13 years old to get a digital Jässäri. This is because of the user terms and conditions of the Helsinki Profile. You can get a paper application form from the Youth Centres of the City. ",
+    "linkForMembersText": "Do you have membership? Log in",
+    "registrationForm": "http://jassari.munstadi.fi/files/2019/06/Jasenkorttikaavake_2019_englanti.pdf",
+    "registrationFormText": "Download application",
+    "return": "Go back",
+    "title": "Get your Jässäri!"
   },
+  "Fill in the application form with your guardian and take it back to the Youth Centre to get your own membership card. ": "",
   "membershipDetails": {
-    "title": "Own information",
-    "text": "Here information about things?",
-    "profileInformation": "Basic information",
     "additionalInformation": "Additional information",
-    "requiredPermissionsFromParent": "Required permissions from parent",
+    "edit": "Edit information",
     "photoUsageExplanation": "Pictures of my child can be taken and used for internal and external communications in the City of Helsinki",
+    "profileInformation": "Basic information",
+    "requiredPermissionsFromParent": "Required permissions from parent",
     "returnToFront": "Return to frontpage",
-    "edit": "Edit information"
+    "text": "Here information about things?",
+    "title": "Own information"
   },
   "membershipInformation": {
-    "title": "Membership card number {{number}}",
-    "pageTitle": "Membership information",
-    "showProfileInformation": "Show profile information",
-    "validUntil": "Valid until {{date}}",
     "renew": "Renew membership",
+    "renewSuccessMessage": "Membership renewed successfully!",
     "renewSuccessTitle": "Success",
-    "renewSuccessMessage": "Membership renewed successfully!"
+    "showProfileInformation": "Show profile information",
+    "title": "Membership card number {{number}}",
+    "validUntil": "Valid until {{date}}"
+  },
+  "nav": {
+    "menuButtonLabel": "Profile menu",
+    "profileLinkTest": "https://profiili.test.kuva.hel.ninja/",
```

Comment:
I would personally rather take value from the environment, so that we easily can have all kinds of environments: develop, review, staging, test, pre-production, whatever comes to mind..

Reply:
That is indeed much better place for it!